spa/src/analytics/proxy-analytics-client.ts

```diff
@@ -0,0 +1,34 @@
+import { AnalyticClient, ScreenEventProps, TrackEventProps, UIEventProps } from "./types";
+import { axiosRest } from "../api/axiosInstance";
+
+export const proxyAnalyticsClient = function(): AnalyticClient {
+
+	const sendAnalytics = (eventType: string, eventProperties: ScreenEventProps | UIEventProps | TrackEventProps, eventAttributes?: Record<string, unknown>) => {
+		AP.context.getToken((jwt: string) => {
```

Comment:
TODO: might need some caching here so that we dont' bombard jira's server. Unless the cache is already done in side the `Ap.context.getToken`?

Reply:
I think it is generated in client side, haven't seen any calls to backend
<img width="229" alt="image" src="https://github.com/atlassian/github-for-jira/assets/20631664/84fa64cd-3bf7-4a14-81b6-77c718fd6561">